### ISSUE TITLE
All font sizes should be defined in rems in the default theme

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#5188]
As part of our accessibility audit, we were encouraged to abandon the use of `px` to define font sizes. I'd prefer we use `rem`, and we should be able to get rid of most cases in the default theme just by editing the `variables.less` file.

**Comment by NateWr** [comment]
All line heights should be `em`.

**Comment by NateWr** [comment]
Fixed as part of work in https://github.com/pkp/pkp-lib/issues/5187

**Comment by NateWr** [comment]
@alvgalrus Ahh, sorry to hear that! I don't think that we will revert this because it was a recommendation from our accessibility audit. If it helps, `1rem` = `16px`. So you can try to convert the equation to `rem` or consider using `calc()` (although you should beware of the [known issues](https://caniuse.com/calc)).
